.changeset/flat-guests-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+Gas Estimator L1GasOracle refactor
```

Comment:
Could you also include that we're deprecating the arbgas cmd as a second point in here? Even though it hasn't really been used recently, might be a good to call out anyways